composer.json

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "ubmannheim/publist4ubma2",
```

Comment:
Yes, I think "unima" would be better, because ubmannheim belongs to unima in my opinion.
Thank you for the composer, haven't heard about it jet.